--- ve/lib/python2.7/site-packages/staticjinja/staticjinja.py
+++ ve/lib/python2.7/site-packages/staticjinja/staticjinja.py
@@ -360,10 +360,11 @@
     # Coerce search to an absolute path if it is not already
     if not os.path.isabs(searchpath):
         # TODO: Determine if there is a better way to write do this
-        calling_module = inspect.getmodule(inspect.stack()[-1][0])
+        #calling_module = inspect.getmodule(inspect.stack()[-1][0])
         # Absolute path to project
-        project_path = os.path.realpath(os.path.dirname(
-            calling_module.__file__))
+        #project_path = os.path.realpath(os.path.dirname(
+        #    calling_module.__file__))
+        project_path = os.path.realpath(os.curdir)
         searchpath = os.path.join(project_path, searchpath)
 
     loader = FileSystemLoader(searchpath=searchpath,
